README.md

```diff
@@ -49,70 +79,135 @@ const client = new Realtime(ABLY_API_KEY);
 const spaces = new Spaces(client);
```

Comment:
Should we update this with the alternative instantiation, with no client?:

Using just the key:
```js
const spaces = new Spaces(ABLY_API_KEY);
```

Using client options:
```js
const spaces = new Spaces({ key: ABLY_API_KEY });
```

These are nice for just hacking away. The client instance, if needed for pub/sub or anything else, can then be accessed via `spaces.ably`.

Reply:
I didn't realise you could instantiate like this(!) That's much cleaner so have updated the examples.